### PR TITLE
[Keyboard] Add VIA support to v60 Type R

### PR DESCRIPTION
**Pull request by digitaltoad**
I've added VIA compatibility to the v60 Type R as well as created a unique PRODUCT_ID for the v60 Type R in accordance with VIA's requirements.

## Description

* Updated PRODUCT_ID to a value that is globally unique in the source related to PRODUCT_IDs in accordance with VIA [documentation](https://caniusevia.com/docs/configuring_qmk#change-vendor_id-product_id)
* Updated the PRODUCT to include the manufacturer in accordance with VIA [documentation](https://caniusevia.com/docs/configuring_qmk#change-product)
* Added a via keymap using the default keymap.c and updated this keymaps rules to enable VIA
* Enabled bootmagic lite as recommended by VIA [documentation](https://caniusevia.com/docs/configuring_qmk#changes-to-keyboard-directorys-rulesmk)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/v60_type_r/config.h [hunk 1]
```diff
@@ -21,10 +21,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6060
+#define PRODUCT_ID      0x0658
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    KB Paradise
-#define PRODUCT         V60 Type R
+#define PRODUCT         KB Paradise V60 Type R
```
Comment (fauxpark) [inline keyboards/v60_type_r/config.h:27]:
```suggestion
#define PRODUCT         V60 Type R
```

File: keyboards/v60_type_r/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   * |LShift  |  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /| RShift   |
+   * |-----------------------------------------------------------|
+   * |Ctrl|Gui |Alt |      Space              |Fn0 |Gui |App|Ctrl|
+   * `-----------------------------------------------------------'
+   */
+  [0] = LAYOUT_60_ansi(
+	       KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, \
+	       KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, \
+	       KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  \
+	       KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,   \
```
Comment (fauxpark) [inline keyboards/v60_type_r/keymaps/via/keymap.c:41]:
```suggestion
	       KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,
	       KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,
	       KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,
	       KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,
```

File: keyboards/v60_type_r/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+   * |        |   |   |   |   |VolD|VolU|Mut|End|PgD|Dwn|        |
+   * |-----------------------------------------------------------|
+   * |    |    |    |                        |    |    |    |    |
+   * `-----------------------------------------------------------'
+   */
+  [1] = LAYOUT_60_ansi(
+	       KC_GRV,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_DEL, \
+	       KC_TRNS,   KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,    KC_TRNS,          KC_INS, \
+	       KC_TRNS,   KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_HOME, KC_PGUP, KC_LEFT, KC_RIGHT,          KC_TRNS,          \
+	       KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_VOLD,   KC_VOLU, KC_MUTE, KC_END,  KC_PGDN,  KC_DOWN,           KC_TRNS,  \
```
Comment (fauxpark) [inline keyboards/v60_type_r/keymaps/via/keymap.c:61]:
```suggestion
	       KC_GRV,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_DEL,
	       KC_TRNS,   KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,    KC_TRNS,          KC_INS,
	       KC_TRNS,   KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_HOME, KC_PGUP, KC_LEFT, KC_RIGHT,          KC_TRNS,
	       KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_VOLD,   KC_VOLU, KC_MUTE, KC_END,  KC_PGDN,  KC_DOWN,           KC_TRNS,
```

File: keyboards/v60_type_r/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	       KC_TRNS,   KC_TRNS,  KC_TRNS,            KC_TRNS,                                                           KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS),
+
+};
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
Comment (fauxpark) [inline keyboards/v60_type_r/keymaps/via/keymap.c:68]:
```suggestion
```

File: keyboards/v60_type_r/rules.mk [hunk 5]
```diff
@@ -14,7 +14,7 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/v60_type_r/rules.mk:17]:
```suggestion
BOOTMAGIC_ENABLE = no         # Virtual DIP switch configuration
```

File: keyboards/v60_type_r/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+   * |    |    |    |                        |    |    |    |    |
+   * `-----------------------------------------------------------'
+   */
+  [1] = LAYOUT_60_ansi(
+	       KC_GRV,    KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,   KC_F12,  KC_DEL, \
+	       KC_TRNS,   KC_TRNS,  KC_UP,    KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP,    KC_TRNS,          KC_INS, \
+	       KC_TRNS,   KC_LEFT,  KC_DOWN,  KC_RIGHT, KC_TRNS,  KC_TRNS,  KC_TRNS,   KC_TRNS, KC_HOME, KC_PGUP, KC_LEFT, KC_RIGHT,          KC_TRNS,          \
+	       KC_TRNS,   KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_VOLD,   KC_VOLU, KC_MUTE, KC_END,  KC_PGDN,  KC_DOWN,           KC_TRNS,  \
+	       KC_TRNS,   KC_TRNS,  KC_TRNS,            KC_TRNS,                                                           KC_TRNS,  KC_TRNS, KC_TRNS, KC_TRNS),
+
```
Comment (fauxpark) [inline keyboards/v60_type_r/keymaps/via/keymap.c:63]: You need two more empty layers here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/v60_type_r/rules.mk [hunk 7]
```diff
@@ -14,7 +14,7 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = no         # Virtual DIP switch configuration
```
Comment (fauxpark) [inline keyboards/v60_type_r/rules.mk:17]:
```suggestion
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**